components/Downloads/ChangelogModal/index.module.css

```diff
@@ -0,0 +1,98 @@
+.overlay {
+  @apply fixed
+    inset-0
+    flex
+    justify-center
+    bg-white
+    bg-opacity-90
+    backdrop-blur-lg
+    dark:bg-neutral-950
+    dark:bg-opacity-80;
+
+  .content {
+    @apply relative
+      mx-4
+      my-4
+      inline-flex
+      w-full
+      flex-col
+      overflow-scroll
+      rounded
+      border
+      border-neutral-200
+      bg-white
+      p-8
+      focus:outline-none
+      dark:bg-neutral-950
+      sm:mt-20
+      sm:w-1/2
+      sm:p-12;
+  }
+
+  .trigger {
+    @apply absolute
+      right-3
+      top-3
+      block
+      h-6
+      w-6
+      cursor-pointer
+      sm:hidden;
+  }
+
+  .title {
+    @apply mb-2
+      text-3xl
+      font-semibold
+      text-neutral-900
+      dark:text-white;
+  }
+
+  .description {
+    @apply mb-4
+      text-lg
+      font-regular
+      text-neutral-800
+      dark:text-neutral-200;
+  }
+
+  .authors {
+    @apply mb-8
+      flex
+      flex-wrap
+      items-center
+      gap-4;
+
+    a {
+      @apply flex
+        items-center
+        gap-1
+        text-xs
+        text-neutral-600
+        underline
+        visited:text-neutral-600;
+    }
+
+    svg {
+      @apply h-3
+        w-3
+        text-neutral-600;
+    }
+  }
+
+  .wrapper {
+    @apply flex
+      flex-col
+      gap-4;
+
+    h1,
```

Comment:
I was thinking we probably should define these on the base css file for the new styles, otherwise we might have the same issue on other pages. (Probably a new file typography.css) since we also have styling for ul, li, tables, etc